steps/s3/session.go

```diff
@@ -84,24 +84,47 @@ func (s *Session) uploadS3FileWithContent(ctx context.Context, bucket, key, mess
 	return nil
 }
 
-// createBucket creates a new bucket.
-func (s *Session) createBucket(ctx context.Context, bucket string) error {
+// CreateS3Bucket creates a new bucket.
+func (s *Session) CreateS3Bucket(ctx context.Context, bucket string) error {
 	logger := GetLogger()
-	logger.LogMessage(fmt.Sprintf("Creating a new bucket: %s", bucket))
+	logger.LogMessage(fmt.Sprintf("creating a new bucket: %s", bucket))
 	cparams := &s3.CreateBucketInput{
 		Bucket: aws.String(bucket),
 	}
 
 	s3Client := s3.New(s.Client)
 	_, err := s3Client.CreateBucket(cparams)
 	if err != nil {
-		logger.LogMessage(fmt.Sprintf("Error creating a new bucket: %s, err: %v", bucket, err))
+		logger.LogMessage(fmt.Sprintf("error creating a new bucket: %s, err: %v", bucket, err))
 	}
 	return nil
 }
 
-// validateS3File checks the existence of a file in S3.
-func (s *Session) validateS3FileExists(ctx context.Context, bucket, key string) error {
+// ValidateS3Bucket verifies the existence of a bucket.
+func (s *Session) ValidateS3Bucket(ctx context.Context, bucket string) error {
```

Comment:
So these would be:

`- ValidateS3BucketExists`

Reply:
You're right